service-commands/scripts/createSolanaUserBanks.js

```diff
@@ -0,0 +1,398 @@
+const {
+  createUserBankFrom
+} = require('@audius/libs/src/services/solanaWeb3Manager/userBank')
+const SolanaUtils = require('@audius/libs/src/services/solanaWeb3Manager/utils')
+const IdentityService = require('@audius/libs/src/services/identity')
+const axios = require('axios')
+const solanaWeb3 = require('@solana/web3.js')
+const { PublicKey, Keypair } = solanaWeb3
+const config = require('../config/config')
+const untildify = require('untildify')
+const { program } = require('commander')
+const fs = require('fs')
+const LineByLineReader = require('line-by-line')
+const qs = require('qs')
+
+/**
+ * @typedef {Object} SolanaConfig
+ * @property {string} SOLANA_ENDPOINT
+ * @property {string} SOLANA_MINT_ADDRESS
+ * @property {string} SOLANA_TOKEN_ADDRESS
+ * @property {string} SOLANA_CLAIMABLE_TOKEN_PROGRAM_ADDRESS
+ * @property {string} SOLANA_REWARDS_MANAGER_PROGRAM_ID
+ * @property {string} SOLANA_REWARDS_MANAGER_PROGRAM_PDA
+ * @property {string} SOLANA_REWARDS_MANAGER_TOKEN_PDA
+ * @property {string} SOLANA_FEE_PAYER_SECRET_KEY
+ */
+
+/**
+ * @typedef {Object} ServiceConfig
+ * @property {string} url
+ */
+
+/**
+ * @typedef {Object} Config
+ * @property {SolanaConfig} solanaConfig
+ * @property {ServiceConfig} identityServiceConfig
+ * @property {ServiceConfig} discoveryProviderConfig
+ */
+
+/**
+ * Gets a batch of users from the /users endpoint to get their wallets
+ * Preferred over depending on libs since we want to specify a DN manually
```

Comment:
just fyi -- libs can be initialized w/ a singly whitelisted DN, but this is fine